### PR TITLE
[admin[ Updating contributor guide

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR is based on the updates in https://github.com/OfficeDev/office-js-docs-reference/pull/1153. If leaves the existing verbage explaining sample contributions, since the audience is slightly different.

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -1,42 +1,83 @@
...
-* [FAQ](#faq)
-* [More resources](#more-resources)
+* [Report issues or request additional samples.](#report-issues)
+* [Add or edit new samples.](#add-or-edit-samples)
 
-## Ways to contribute
+## Report issues
 
-Here are some ways you can contribute to this documentation:
+Most of this repo's content is generated from [type definition files](https://github.com/DefinitelyTyped/DefinitelyTyped). As such, corrections here are likely to be overwritten. Please report issues or make requests for additional documentation through [GitHub Issues](https://github.com/OfficeDev/office-scripts-docs-reference/issues). To do this, go to the **Feedback** section at the bottom of the affected article, then select **This page** to create a GitHub issue. Alternatively, create a new issue directly on [GitHub](https://github.com/OfficeDev/office-scripts-docs-reference/issues/new).
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:10]: Is this true for Office Scripts that the type definition files are in DefinitelyTyped?

File: Contributing.md [hunk 2]
```diff
@@ -1,42 +1,83 @@
...
+```
 
-**Note**: Most of the information in this section can be found in [GitHub Help](https://docs.github.com/) articles. If you're familiar with Git and GitHub, skip to the [Adding samples](#adding-samples) section for the specifics of the code/content flow of this repository.
+Please add samples to the **.yaml** file in alphabetical order, based on the class and method names. This makes them easier to maintain.
 
-Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. It's the recommended way to add a single sample.
+### What makes a good sample?
 
-> **Important**: The reference content in this repository is autogenerated. Pull requests to this repo should either add samples or address tooling or other infrastructure issues.
+A good sample has the following characteristics:
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:48]: Here and other instances
```suggestion
A good sample has the following characteristics.
```

File: Contributing.md [hunk 3]
```diff
@@ -1,42 +1,83 @@
...
+### Contribute using GitHub
+
+Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. Use this method to make a minor change that doesn't involve code changes.
+
+Using this method allows you to contribute to one article at a time.
+
+#### To contribute using GitHub
+
+1. Find the article you want to contribute to on GitHub.
 2. Once you are on the article in GitHub, sign in to GitHub (get a free account [Join GitHub](https://github.com/join)).
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:68]: nit: for consistency, can all the ordered lists be formatted similarly (e.g. all 1's or 123)?

File: Contributing.md [hunk 4]
```diff
@@ -88,17 +131,17 @@ To make the contribution process as seamless as possible, follow these steps.
...
 
-You navigate to the repository on your computer by using File Explorer. The repository files are in `C:\Users\<yourusername>\<repo name>`.
+You navigate to the repository on your computer by using File Explorer. The repository files are in `C:\Users\<yourusername>\office-scripts-docs-reference`.
 
 To edit files, open them in an editor of your choice and modify them. To create a new file, use the editor of your choice and save the new file in the appropriate location in your local copy of the repository. While working, save your work frequently.
 
-The files in `C:\Users\<yourusername>\<repo name>` are a working copy of the new branch that you created in your local repository. Changing anything in this folder doesn't affect the local repository until you commit a change. To commit a change to the local repository, type the following commands in GitBash:
+The files in `C:\Users\<yourusername>\office-scripts-docs-reference` are a working copy of the new branch that you created in your local repository. Changing anything in this folder doesn't affect the local repository until you commit a change. To commit a change to the local repository, type the following commands in GitBash:
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:144]:
```suggestion
The files in `C:\Users\<yourusername>\office-scripts-docs-reference` are a working copy of the new branch that you created in your local repository. Changing anything in this folder doesn't affect the local repository until you commit a change. To commit a change to the local repository, type the following commands in GitBash.
```

File: Contributing.md [hunk 5]
```diff
@@ -118,21 +161,21 @@ When you're finished with your work and are ready to have it merged into the mai
...
 
 #### Create a new branch after merge
 
 After a branch is successfully merged (that is, your pull request is accepted), don't continue working in that local branch. This can lead to merge conflicts if you submit another pull request. To do another update, create a new local branch from the successfully merged upstream branch, and then delete your initial local branch.
 
-For example, if your local branch X was successfully merged into the OfficeDev/office-scripts-docs-reference master branch and you want to make additional updates to the content that was merged. Create a new local branch, X2, from the OfficeDev/office-scripts-docs-reference master branch. To do this, open GitBash and execute the following commands:
+For example, if your local branch X was successfully merged into the OfficeDev/office-scripts-docs-reference main branch and you want to make additional updates to the content that was merged. Create a new local branch, X2, from the OfficeDev/office-scripts-docs-reference main branch. To do this, open GitBash and execute the following commands:
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:175]:
```suggestion
For example, if your local branch X was successfully merged into the OfficeDev/office-scripts-docs-reference main branch and you want to make additional updates to the content that was merged. Create a new local branch, X2, from the OfficeDev/office-scripts-docs-reference main branch. To do this, open GitBash and execute the following commands.
```

File: Contributing.md [hunk 6]
```diff
@@ -1,42 +1,83 @@
...
+
+### Adding a sample script to the GitHub repository
+
+All the example code in this repository comes from one of the base sample **.yaml** files. The Office Scripts for Excel samples are in **[/docs/sample-scripts/excel-scripts.yaml](/docs/sample-scripts/excel-scripts.yaml)**. Use one of the methods described earlier in this guide to edit the file. You can also create an issue with the desired sample code and we'll add it to the documentation.
+
+Samples are tied to particular APIs. Pick the API that your sample showcases and use it as the YAML key. A YAML key for the Excel sample file has this format: `'ExcelScript.'`*\<class-name\>*`#`*\<method-name\>*`:member(`*\<overload-number\>*`)`. The *\<overload-number\>* indicates which method is being referenced when there are multiple options. The value is usually `1`.
+
+After the YAML key, add `:`, then add `  - |-` on a new line. This ensures the spacing in the example stays as is.
+
+The following example shows a sample script for `Range.getValue`:
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:24]:
```suggestion
The following example shows a sample script for `Range.getValue`.
```

File: Contributing.md (hunk 1 above)
Comment (AlexJerabek) [reply Contributing.md:10]: Good catch. They are not.

File: Contributing.md [hunk 7]
```diff
@@ -1,42 +1,83 @@
...
-2. Once you are on the article in GitHub, sign in to GitHub (get a free account [Join GitHub](https://github.com/join)).
-3. Choose the **pencil icon** (edit the file in your fork of this project) and make your changes in the **<>Edit file** window.
-4. Scroll to the bottom and enter a description.
-5. Choose **Propose file change**>**Create pull request**.
+Using this method allows you to contribute to one article at a time.
+
+#### To contribute using GitHub
+
+1. Find the article you want to contribute to on GitHub.
+1. Once you are on the article in GitHub, sign in to GitHub (get a free account [Join GitHub](https://github.com/join)).
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:68]:
```suggestion
1. Once you are on the article in GitHub, sign in to GitHub (to get a free account, [Join GitHub](https://github.com/join)).
```

File: Contributing.md [hunk 8]
```diff
@@ -49,7 +90,9 @@ As a community member, you must sign the Contribution License Agreement (CLA) be
 
 Signing the CLA does not grant you rights to commit to the main repository, but it does mean that the Office Developer and Office Developer Content Publishing teams will be able to review and approve your contributions. You are credited for your submissions.
 
-Pull requests are typically reviewed within 10 business days.
+## Use GitHub, Git, and this repository
+
+**Note:** Most of the information in this section can be found in [GitHub Help] articles. If you're familiar with Git and GitHub, skip to the **Contribute and edit content** section for the specifics of the code/content flow of this repository.
```
Comment (ElizabethSamuel-MSFT) [inline Contributing.md:95]:
```suggestion
**Note:** Most of the information in this section can be found in [GitHub Help] articles. If you're familiar with Git and GitHub, skip to the [Contribute and edit content](#contribute-and-edit-content) section for the specifics of the code/content flow of this repository.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks good. Just had a couple more suggestions.

**Pull request merged**